localstack/runtime/analytics.py

```diff
@@ -20,8 +20,10 @@
     "KINESIS_PROVIDER",
     "KMS_PROVIDER",
     "LAMBDA_DOWNLOAD_AWS_LAYERS",
+    # Irrelevant post v3 but intentionally tracked for some time
     "LAMBDA_EXECUTOR",
```

Comment:
It might be interesting to know how many users still have legacy variables configured post v3 🤷

Reply:
can't hurt and we can still turn it off later :+1: